elastic/kibana
# [Synthetics] SLO Availability sync delay field to use timestamp instead of event.ingested !!

## Summary

Fixes https://github.com/elastic/kibana/issues/196548

SLO Availability sync delay field to use `@timestamp` instead of `event.ingested`  !!


### Testing

- Make sure Synthetics availability SLOs works as expected in serverless and stateful 
- Make sure when SLO is updated, it continues to work
